### DISCUSSION TITLE
Can I change my plots key?

### DISCUSSION THREAD

**Discussion by DexterXR** [Chia-Network/chia-blockchain#4921] [Q&A]
Hi! I have some plots in my old machine, and I use them on HPool.
Now I stop with Hpool and farm solo, generating a new key.

The problem is, my old plots can´t sync with my new key.

Is possible change de plots key ou import this plots to my new key?

Thank you!

**Comment by mptm436** [comment]
same situation here, now i have two keys and two wallets, if the plots can not be modified, can i farm with two keys on the same machine
